src/pages/PlanPage/subRoutes/UpgradePlanPage/UpgradeForm/PlanTypeOptions/PlanTypeOptions.tsx

```diff
@@ -47,7 +45,8 @@ const PlanTypeOptions: React.FC<PlanTypeOptionsProps> = ({
   })
 
   const hasTeamPlans = shouldDisplayTeamCard({ plans })
-  const plan = accountDetails?.rootOrganization?.plan ?? accountDetails?.plan
+  const plan =
+    accountDetails?.rootOrganization?.plan?.value ?? accountDetails?.plan?.value
```

Comment:
this was also a bug :)

Reply:
(same question as above if you don't mind)